src/js/models/maps/assets/CesiumVectorData.js

```diff
@@ -603,12 +606,20 @@ define([
        * @since 2.25.0
        */
       styleBillboard: function (entity, styles) {
-        if (!this.pinBuilder) {
-          this.pinBuilder = new Cesium.PinBuilder();
-        }
-        entity.billboard.image = this.pinBuilder
-          .fromColor(styles.color, styles.markerSize)
-          .toDataURL();
+        const pin = new DOMParser()
```

Comment:
I'm no SVG expert, but could you just set these attributes on the string defined on line 18?

```
const PIN_SVG_STRING = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 384 512'><!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d='M215.7 499.2C267 435 384 279.4 384 192C384 86 298 0 192 0S0 86 0 192c0 87.4 117 243 168.3 307.2c12.3 15.3 35.1 15.3 47.4 0zM192 128a64 64 0 1 1 0 128 64 64 0 1 1 0-128z'
stroke='white' fill-rule='evenodd'></svg>";
```

As I type out that example, I realize that some of these attributes are dynamic, so maybe can we just move up the ones that are hardcoded?

Reply:
The svg string was copied straight from fontawesome, I think there's benefit to keep our customization separate from the copied string. I'll add a comment to indicate where the string is from.